apps/omg/lib/omg/application.ex

```diff
@@ -20,9 +20,11 @@ defmodule OMG.Application do
 
   use Application
   alias OMG.Alert.AlarmHandler
+  require Logger
 
   def start(_type, _args) do
     :ok = AlarmHandler.install()
+    {:ok, _} = Logger.add_backend(Sentry.LoggerBackend)
```

Comment:
move this option to `config/config.exs`
```elixir
 config :logger,
    backends: [Sentry.LoggerBackend, :console]
```
tests will pass, and backend will be available even application is restarted.